src/components/Editor/Runners/PythonRunner/VisualOutputPane.jsx

```diff
@@ -56,10 +59,14 @@ const VisualOutputPane = () => {
       p5Output.current &&
       p5Output.current.innerHTML !== ""
     ) {
-      if (Sk.py5.stop) {
+      if (Sk.py5?.stop) {
```

Comment:
given we're assigning this function to `window.p5` or `window.py5` now, do we need to be checking `Sk.p5` and `Sk.py5` at all anymore?

Reply:
nice 👍 good spot, have removed 😄